### DISCUSSION TITLE
PointCloudFromArray with Size Input

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#487] [Feature Requests] [answered]
as the title says, i'd like to control the size of the points individually but don't find the right spot in the code to do this.

**Comment by pandrr** [comment, marked as answer]
yeah i wonder how we never had this...

i added it, send you an invite...

in general: you have to set an attribute and read it in the vertex shader where glPointSize is set. has to go into the whole existing calculation in pointmaterial then
